src/Psalm/Storage/ClassConstantStorage.php

```diff
@@ -60,7 +60,6 @@ class ClassConstantStorage
 
     /**
      * @var list<AttributeStorage>
-     * @psalm-suppress PossiblyUnusedProperty
```

Comment:
This actually seems like a false negative that was introduced somewhere.

Reply:
Actually I guess it's just 08413d7, and the reason it's not a false negative is because it's not marked internal.